packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -204,26 +205,42 @@ class _AuthenticatorState extends State<Authenticator> {
   final AuthService _authService = AmplifyAuthService();
 
   late final StateMachineBloc _stateMachineBloc;
+  AmplifyConfig? _config;
 
   @override
   void initState() {
     super.initState();
     _stateMachineBloc = StateMachineBloc(_authService)
       ..authEvent.add(GetCurrentUser());
+    waitForConfiguration().then((value) {
```

Comment:
Do we need the added stream controller? 

```dart
Amplify.configCompleter.future
  .then((config) {
    if (mounted) {
      setState(() => _config = config);
    }
  })
  .catchError((e) => show error or something ?);
```

Reply:
I thought this could be useful in the future if we introduce the ability to call 'configure' multiple times, but no - we don't strictly need it. I'll remove for now.

Reply:
Ah gotcha - yeah, we did discuss that. My bad! Fine either way, but yeah, I still prefer introducing complexity when needed.